noir-projects/noir-contracts/contracts/test_contract/src/main.nr

```diff
@@ -334,7 +341,7 @@ contract Test {
     #[aztec(private)]
     fn set_constant(value: Field) {
         let mut note = TestNote::new(value);
-        storage.example_constant.initialize(&mut note, false);
+        storage.example_constant.initialize(&mut note, false, GrumpkinPoint::zero());
```

Comment:
Passing the zero should fail as we try to derive the `ivpk_app` through addition which would fail as the point is not on the curve. The current setup is not doing the logs that way, which is why it is passing here, but it could be a real pain 😬 , tagging #6408 as this will where we see the pain.